core/spec/lib/tasks/migrations/create_vat_prices_spec.rb

```
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe 'solidus:migrations:create_vat_prices' do
+  describe 'up' do
+    include_context(
+      'rake',
+      task_path: Spree::Core::Engine.root.join('lib/tasks/migrations/create_vat_prices.rake'),
+      task_name: 'solidus:migrations:create_vat_prices:up',
+    )
+
+    # TODO: Update to work if this is missing, and add more specs
```

If that zone is missing, the task should silently return.